Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
+
+COPY . /app
+WORKDIR /app
+RUN pip install -e .
+
+ENV XSNIPPET_API_SETTINGS=/app/xsnippet_api/default.conf
```

Comment:
Indeed. I just wanted to have an env variable for documentation purposes and looks like does not allow to leave it undefined.